src/types.ts

```diff
@@ -18,6 +18,10 @@ export interface StorageMeta {
 // TODO: type ttl
 export type TransactionOptions = Record<string, any>;
 
+export type GetKeysOptions = TransactionOptions & {
+  depth?: number;
```

Comment:
Thinking more, perhaps `maxDepth` is more readable?

Reply:
I agree, I already renamed the function parameter for the same reason

I'll update